application/aspnet/sdk-example/README.md

```diff
@@ -0,0 +1,37 @@
+<p align="center">
+  <img src="https://secrethub.io/img/integrations/aspnet/github-banner.png?v1" alt="ASP.NET + SecretHub" height="230">
+</p>
+<br/>
+
+This example aims to display the use of the native .NET SecretHub integration.  Following the below steps will result in an welcome message on http://localhost:5000. If any error shall occur, you will receive an appropriate descriptive message at the same address.
```

Comment:
```suggestion
This example demonstrates the use of the [native SecretHub .NET integration](https://github.com/secrethub/secrethub-dotnet) in an ASP.NET application.
Following the steps below will result in an welcome message on http://localhost:5000.
If any error occurs, you will receive a descriptive error message in the console.
```

Reply:
I'm also not sure about calling it "the native SecretHub .NET integration". Maybe calling it "the SecretHub .NET client" would be better...

What do you think, @florisvdg?

Reply:
We called it .NET SDK in the guide.

Reply:
Would you liked it changed from "native .NET SecretHub integration" to ".NET SDK"?